src/dac_model.h

```diff
@@ -89,16 +89,6 @@ void assign_to_audio_encoder(dac_model * model, std::string name, ggml_tensor *
 // the context used for running the dac model
 struct dac_context : runner_context {
     dac_context(dac_model * model, int n_threads): runner_context(n_threads), model(model) {};
-    ~dac_context() {
-        ggml_backend_sched_free(sched);
-        ggml_backend_free(backend_cpu);
-        if (backend) {
-            ggml_backend_free(backend);
-        }
-        if (buf_output) {
-            ggml_backend_buffer_free(buf_output);
-        }
-    }
```

Comment:
I'll need to test these deletions; I introduced them while hunting down a memory leak, so I'd like to confirm

Reply:
I *moved* these to `runner_context::virtual ~runner_context` so that nobody forgets my new `ggml_threadpool_free(threadpool)`. A derived destructor will call the base destructor.

Reply:
Sorry didn't see that.